### DISCUSSION TITLE
[Question] Differences between Metamod-R and Metamod-P

### DISCUSSION THREAD

**Discussion by rtxa** [theAsmodai/metamod-r#83] [Q&A] [answered]
Hi, I wonder what difference has Metamod-R with Metamod-P besides only offering ReHLDS support? Does Metamod-R has some new feature or some new API functions?

**Comment by stamepicmorg** [comment, marked as answer]
Metamod-r is based on the original version of [Metamod](http://metamod.org/) written by Will Day for Half-Life 1 with some improvements from [Jussi Kivilinna](https://github.com/jkivilin) ([Metamod-p](https://github.com/jkivilin/metamod-p)). This product contains a large number of performance optimizations and more pure code. The core was written using JIT compiler.

Metamod-r is recommended to be run with [ReHLDS](https://github.com/dreamstalker/ReHLDS) (API 3.1+). There is compatible with original HLDS
